src/style.css

```diff
@@ -1,3 +1,110 @@
+:root {
+  font-family: Roboto, sans-serif;
```

Comment:
2 [CHECKLIST] - Remember to use fallback fonts - alternative font-family in case the main one doesn't work [like this](https://www.w3schools.com/cssref/pr_font_font-family.asp)